packages/sources/src/components/label-group/label-group.scss

```diff
@@ -2,7 +2,9 @@
   display: flex;
   flex-wrap: wrap;
   gap: $u1;
+  list-style: none;
```

Comment:
![image](https://user-images.githubusercontent.com/9696261/188843429-efebd0be-3cbf-4d80-937d-e82a000a94c6.png)

Ruik ik daar... 🍰 ?

Reply:
Dit is er wel eentje voor in de "ga je schamen" boeken inderdaad...

Reply:
Doe maar 'n aardbeienslof en 'n worteltaart en we hebben het er niet meer over.

cc @gofix @smvandermeulen @LukeVerlaan @PieterVerheij @svrijsewijk